OpenRA.Game/Map/MapGrid.cs

```diff
@@ -79,6 +79,47 @@ public class MapGrid : IGlobalModData
 			new[] { 0, 1, 0, 1 }
 		};
 
+		readonly WRot[] slopes = new WRot[]
+		{
+			// Flat
+			new WRot(WAngle.Zero, WAngle.Zero, WAngle.Zero),		// 0
+
+			// Slopes (two corners high)
+			new WRot(new WAngle(-76), new WAngle(-40), WAngle.Zero),	// 1
+			new WRot(new WAngle(76), new WAngle(-40), WAngle.Zero),	// 2
+			new WRot(new WAngle(76), new WAngle(40), WAngle.Zero),	// 3
+			new WRot(new WAngle(-76), new WAngle(40), WAngle.Zero),	// 4
+
+			// Slopes (one corner high)
+			new WRot(WAngle.Zero, new WAngle(-76), WAngle.Zero),	// 5
+			new WRot(new WAngle(76), WAngle.Zero, WAngle.Zero),	// 6
+			new WRot(WAngle.Zero, new WAngle(76), WAngle.Zero),	// 7
+			new WRot(new WAngle(-76), WAngle.Zero, WAngle.Zero),	// 8
+
+			// Slopes (three corners high)
+			new WRot(WAngle.Zero, new WAngle(-76), WAngle.Zero),	// 9
+			new WRot(new WAngle(76), WAngle.Zero, WAngle.Zero),	// 10
+			new WRot(WAngle.Zero, new WAngle(76), WAngle.Zero),	// 11
+			new WRot(new WAngle(-76), WAngle.Zero, WAngle.Zero),	// 12
```

Comment:
Can you make this configurable in `AffectedBySlope` instead?

Reply:
I will ask Castle about it. Later he will fix something.

Reply:
Again, thanks for all the advice here.

Reply:
These angles are related to the other hardcoded offsets, so we don't gain anything by making them configurable.